2021-07-26 03:30:58 Error: [MissingControllerException] Controller class TestsController could not be found.
Exception Attributes: array (
  'class' => 'TestsController',
  'plugin' => NULL,
)
Request URL: /rankings/tests
Stack Trace:
#0 C:\xampp\htdocs\rankings\index.php(117): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#1 {main}
2021-07-26 03:38:32 Error: [MissingActionException] Action LutadoresController::index() could not be found.
Exception Attributes: array (
  'controller' => 'LutadoresController',
  'action' => 'index',
)
Request URL: /rankings/lutadores
Stack Trace:
#0 C:\xampp\cake\lib\Cake\Routing\Dispatcher.php(193): Controller->invokeAction(Object(CakeRequest))
#1 C:\xampp\cake\lib\Cake\Routing\Dispatcher.php(167): Dispatcher->_invoke(Object(LutadoresController), Object(CakeRequest))
#2 C:\xampp\htdocs\rankings\index.php(117): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#3 {main}
2021-07-26 03:40:13 Error: [MissingViewException] View file "Lutadores\index.ctp" is missing.
Exception Attributes: array (
  'file' => 'Lutadores\\index.ctp',
)
Request URL: /rankings/lutadores
Stack Trace:
#0 C:\xampp\cake\lib\Cake\View\View.php(470): View->_getViewFileName('Lutadores\\index')
#1 C:\xampp\cake\lib\Cake\Controller\Controller.php(968): View->render(NULL, NULL)
#2 C:\xampp\cake\lib\Cake\Routing\Dispatcher.php(200): Controller->render()
#3 C:\xampp\cake\lib\Cake\Routing\Dispatcher.php(167): Dispatcher->_invoke(Object(LutadoresController), Object(CakeRequest))
#4 C:\xampp\htdocs\rankings\index.php(117): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#5 {main}
2021-07-27 02:49:27 Error: [Error] Call to a member function find() on null
Request URL: /rankings/lutadores
Stack Trace:
#0 [internal function]: LutadoresController->index()
#1 C:\xampp\cake\lib\Cake\Controller\Controller.php(499): ReflectionMethod->invokeArgs(Object(LutadoresController), Array)
#2 C:\xampp\cake\lib\Cake\Routing\Dispatcher.php(193): Controller->invokeAction(Object(CakeRequest))
#3 C:\xampp\cake\lib\Cake\Routing\Dispatcher.php(167): Dispatcher->_invoke(Object(LutadoresController), Object(CakeRequest))
#4 C:\xampp\htdocs\rankings\index.php(117): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#5 {main}
2021-07-27 02:49:38 Error: [Error] Call to a member function find() on null
Request URL: /rankings/lutadores
Stack Trace:
#0 [internal function]: LutadoresController->index()
#1 C:\xampp\cake\lib\Cake\Controller\Controller.php(499): ReflectionMethod->invokeArgs(Object(LutadoresController), Array)
#2 C:\xampp\cake\lib\Cake\Routing\Dispatcher.php(193): Controller->invokeAction(Object(CakeRequest))
#3 C:\xampp\cake\lib\Cake\Routing\Dispatcher.php(167): Dispatcher->_invoke(Object(LutadoresController), Object(CakeRequest))
#4 C:\xampp\htdocs\rankings\index.php(117): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#5 {main}
2021-07-27 02:52:40 Error: [Error] Call to a member function find() on null
Request URL: /rankings/lutadores
Stack Trace:
#0 [internal function]: LutadoresController->index()
#1 C:\xampp\cake\lib\Cake\Controller\Controller.php(499): ReflectionMethod->invokeArgs(Object(LutadoresController), Array)
#2 C:\xampp\cake\lib\Cake\Routing\Dispatcher.php(193): Controller->invokeAction(Object(CakeRequest))
#3 C:\xampp\cake\lib\Cake\Routing\Dispatcher.php(167): Dispatcher->_invoke(Object(LutadoresController), Object(CakeRequest))
#4 C:\xampp\htdocs\rankings\index.php(117): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#5 {main}
2021-07-27 02:56:33 Error: [Error] Call to a member function find() on null
Request URL: /rankings/lutadores
Stack Trace:
#0 [internal function]: LutadoresController->index()
#1 C:\xampp\cake\lib\Cake\Controller\Controller.php(499): ReflectionMethod->invokeArgs(Object(LutadoresController), Array)
#2 C:\xampp\cake\lib\Cake\Routing\Dispatcher.php(193): Controller->invokeAction(Object(CakeRequest))
#3 C:\xampp\cake\lib\Cake\Routing\Dispatcher.php(167): Dispatcher->_invoke(Object(LutadoresController), Object(CakeRequest))
#4 C:\xampp\htdocs\rankings\index.php(117): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#5 {main}
2021-07-27 02:56:34 Error: [Error] Call to a member function find() on null
Request URL: /rankings/lutadores
Stack Trace:
#0 [internal function]: LutadoresController->index()
#1 C:\xampp\cake\lib\Cake\Controller\Controller.php(499): ReflectionMethod->invokeArgs(Object(LutadoresController), Array)
#2 C:\xampp\cake\lib\Cake\Routing\Dispatcher.php(193): Controller->invokeAction(Object(CakeRequest))
#3 C:\xampp\cake\lib\Cake\Routing\Dispatcher.php(167): Dispatcher->_invoke(Object(LutadoresController), Object(CakeRequest))
#4 C:\xampp\htdocs\rankings\index.php(117): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#5 {main}
2021-07-27 02:56:35 Error: [Error] Call to a member function find() on null
Request URL: /rankings/lutadores
Stack Trace:
#0 [internal function]: LutadoresController->index()
#1 C:\xampp\cake\lib\Cake\Controller\Controller.php(499): ReflectionMethod->invokeArgs(Object(LutadoresController), Array)
#2 C:\xampp\cake\lib\Cake\Routing\Dispatcher.php(193): Controller->invokeAction(Object(CakeRequest))
#3 C:\xampp\cake\lib\Cake\Routing\Dispatcher.php(167): Dispatcher->_invoke(Object(LutadoresController), Object(CakeRequest))
#4 C:\xampp\htdocs\rankings\index.php(117): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#5 {main}
2021-07-27 02:56:35 Error: [Error] Call to a member function find() on null
Request URL: /rankings/lutadores
Stack Trace:
#0 [internal function]: LutadoresController->index()
#1 C:\xampp\cake\lib\Cake\Controller\Controller.php(499): ReflectionMethod->invokeArgs(Object(LutadoresController), Array)
#2 C:\xampp\cake\lib\Cake\Routing\Dispatcher.php(193): Controller->invokeAction(Object(CakeRequest))
#3 C:\xampp\cake\lib\Cake\Routing\Dispatcher.php(167): Dispatcher->_invoke(Object(LutadoresController), Object(CakeRequest))
#4 C:\xampp\htdocs\rankings\index.php(117): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#5 {main}
2021-07-27 02:56:36 Error: [Error] Call to a member function find() on null
Request URL: /rankings/lutadores
Stack Trace:
#0 [internal function]: LutadoresController->index()
#1 C:\xampp\cake\lib\Cake\Controller\Controller.php(499): ReflectionMethod->invokeArgs(Object(LutadoresController), Array)
#2 C:\xampp\cake\lib\Cake\Routing\Dispatcher.php(193): Controller->invokeAction(Object(CakeRequest))
#3 C:\xampp\cake\lib\Cake\Routing\Dispatcher.php(167): Dispatcher->_invoke(Object(LutadoresController), Object(CakeRequest))
#4 C:\xampp\htdocs\rankings\index.php(117): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#5 {main}
2021-07-27 03:00:04 Error: [Error] Call to a member function find() on null
Request URL: /rankings/lutadores
Stack Trace:
#0 [internal function]: LutadoresController->index()
#1 C:\xampp\cake\lib\Cake\Controller\Controller.php(499): ReflectionMethod->invokeArgs(Object(LutadoresController), Array)
#2 C:\xampp\cake\lib\Cake\Routing\Dispatcher.php(193): Controller->invokeAction(Object(CakeRequest))
#3 C:\xampp\cake\lib\Cake\Routing\Dispatcher.php(167): Dispatcher->_invoke(Object(LutadoresController), Object(CakeRequest))
#4 C:\xampp\htdocs\rankings\index.php(117): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#5 {main}
2021-07-27 03:07:59 Error: [Error] Call to a member function find() on null
Request URL: /rankings/lutadores
Stack Trace:
#0 [internal function]: LutadoresController->index()
#1 C:\xampp\cake\lib\Cake\Controller\Controller.php(499): ReflectionMethod->invokeArgs(Object(LutadoresController), Array)
#2 C:\xampp\cake\lib\Cake\Routing\Dispatcher.php(193): Controller->invokeAction(Object(CakeRequest))
#3 C:\xampp\cake\lib\Cake\Routing\Dispatcher.php(167): Dispatcher->_invoke(Object(LutadoresController), Object(CakeRequest))
#4 C:\xampp\htdocs\rankings\index.php(117): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#5 {main}
2021-07-27 03:08:01 Error: [Error] Call to a member function find() on null
Request URL: /rankings/lutadores
Stack Trace:
#0 [internal function]: LutadoresController->index()
#1 C:\xampp\cake\lib\Cake\Controller\Controller.php(499): ReflectionMethod->invokeArgs(Object(LutadoresController), Array)
#2 C:\xampp\cake\lib\Cake\Routing\Dispatcher.php(193): Controller->invokeAction(Object(CakeRequest))
#3 C:\xampp\cake\lib\Cake\Routing\Dispatcher.php(167): Dispatcher->_invoke(Object(LutadoresController), Object(CakeRequest))
#4 C:\xampp\htdocs\rankings\index.php(117): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#5 {main}
2021-07-27 03:08:04 Error: [Error] Call to a member function find() on null
Request URL: /rankings/lutadores
Stack Trace:
#0 [internal function]: LutadoresController->index()
#1 C:\xampp\cake\lib\Cake\Controller\Controller.php(499): ReflectionMethod->invokeArgs(Object(LutadoresController), Array)
#2 C:\xampp\cake\lib\Cake\Routing\Dispatcher.php(193): Controller->invokeAction(Object(CakeRequest))
#3 C:\xampp\cake\lib\Cake\Routing\Dispatcher.php(167): Dispatcher->_invoke(Object(LutadoresController), Object(CakeRequest))
#4 C:\xampp\htdocs\rankings\index.php(117): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#5 {main}
2021-07-27 03:08:05 Error: [Error] Call to a member function find() on null
Request URL: /rankings/lutadores
Stack Trace:
#0 [internal function]: LutadoresController->index()
#1 C:\xampp\cake\lib\Cake\Controller\Controller.php(499): ReflectionMethod->invokeArgs(Object(LutadoresController), Array)
#2 C:\xampp\cake\lib\Cake\Routing\Dispatcher.php(193): Controller->invokeAction(Object(CakeRequest))
#3 C:\xampp\cake\lib\Cake\Routing\Dispatcher.php(167): Dispatcher->_invoke(Object(LutadoresController), Object(CakeRequest))
#4 C:\xampp\htdocs\rankings\index.php(117): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#5 {main}
2021-07-27 03:16:21 Error: [MissingTableException] Table lutadors for model Lutador was not found in datasource default.
Exception Attributes: array (
  'table' => 'lutadors',
  'class' => 'Lutador',
  'ds' => 'default',
)
Request URL: /rankings/lutadores
Stack Trace:
#0 C:\xampp\cake\lib\Cake\Model\Model.php(3673): Model->setSource('lutadors')
#1 C:\xampp\cake\lib\Cake\Model\Model.php(3040): Model->getDataSource()
#2 C:\xampp\cake\lib\Cake\Model\Model.php(3012): Model->_readDataSource('all', Array)
#3 C:\xampp\apps\rankings\Controller\LutadoresController.php(20): Model->find('all')
#4 [internal function]: LutadoresController->index()
#5 C:\xampp\cake\lib\Cake\Controller\Controller.php(499): ReflectionMethod->invokeArgs(Object(LutadoresController), Array)
#6 C:\xampp\cake\lib\Cake\Routing\Dispatcher.php(193): Controller->invokeAction(Object(CakeRequest))
#7 C:\xampp\cake\lib\Cake\Routing\Dispatcher.php(167): Dispatcher->_invoke(Object(LutadoresController), Object(CakeRequest))
#8 C:\xampp\htdocs\rankings\index.php(117): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#9 {main}
2021-07-27 03:16:59 Error: [MissingViewException] View file "Lutador\index.ctp" is missing.
Exception Attributes: array (
  'file' => 'Lutador\\index.ctp',
)
Request URL: /rankings/lutadores
Stack Trace:
#0 C:\xampp\cake\lib\Cake\View\View.php(470): View->_getViewFileName('Lutador\\index')
#1 C:\xampp\cake\lib\Cake\Controller\Controller.php(968): View->render(NULL, NULL)
#2 C:\xampp\cake\lib\Cake\Routing\Dispatcher.php(200): Controller->render()
#3 C:\xampp\cake\lib\Cake\Routing\Dispatcher.php(167): Dispatcher->_invoke(Object(LutadoresController), Object(CakeRequest))
#4 C:\xampp\htdocs\rankings\index.php(117): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#5 {main}
2021-07-27 03:17:26 Error: [Error] Call to a member function find() on null
Request URL: /rankings/lutadores
Stack Trace:
#0 [internal function]: LutadoresController->index()
#1 C:\xampp\cake\lib\Cake\Controller\Controller.php(499): ReflectionMethod->invokeArgs(Object(LutadoresController), Array)
#2 C:\xampp\cake\lib\Cake\Routing\Dispatcher.php(193): Controller->invokeAction(Object(CakeRequest))
#3 C:\xampp\cake\lib\Cake\Routing\Dispatcher.php(167): Dispatcher->_invoke(Object(LutadoresController), Object(CakeRequest))
#4 C:\xampp\htdocs\rankings\index.php(117): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#5 {main}
2021-07-27 03:17:41 Error: [MissingViewException] View file "Lutador\index.ctp" is missing.
Exception Attributes: array (
  'file' => 'Lutador\\index.ctp',
)
Request URL: /rankings/lutadores
Stack Trace:
#0 C:\xampp\cake\lib\Cake\View\View.php(470): View->_getViewFileName('Lutador\\index')
#1 C:\xampp\cake\lib\Cake\Controller\Controller.php(968): View->render(NULL, NULL)
#2 C:\xampp\cake\lib\Cake\Routing\Dispatcher.php(200): Controller->render()
#3 C:\xampp\cake\lib\Cake\Routing\Dispatcher.php(167): Dispatcher->_invoke(Object(LutadoresController), Object(CakeRequest))
#4 C:\xampp\htdocs\rankings\index.php(117): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#5 {main}
2021-07-27 03:18:09 Error: [Error] Call to a member function find() on null
Request URL: /rankings/lutadores
Stack Trace:
#0 [internal function]: LutadoresController->index()
#1 C:\xampp\cake\lib\Cake\Controller\Controller.php(499): ReflectionMethod->invokeArgs(Object(LutadoresController), Array)
#2 C:\xampp\cake\lib\Cake\Routing\Dispatcher.php(193): Controller->invokeAction(Object(CakeRequest))
#3 C:\xampp\cake\lib\Cake\Routing\Dispatcher.php(167): Dispatcher->_invoke(Object(LutadoresController), Object(CakeRequest))
#4 C:\xampp\htdocs\rankings\index.php(117): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#5 {main}
2021-07-27 03:18:18 Error: [MissingViewException] View file "Lutador\index.ctp" is missing.
Exception Attributes: array (
  'file' => 'Lutador\\index.ctp',
)
Request URL: /rankings/lutadores
Stack Trace:
#0 C:\xampp\cake\lib\Cake\View\View.php(470): View->_getViewFileName('Lutador\\index')
#1 C:\xampp\cake\lib\Cake\Controller\Controller.php(968): View->render(NULL, NULL)
#2 C:\xampp\cake\lib\Cake\Routing\Dispatcher.php(200): Controller->render()
#3 C:\xampp\cake\lib\Cake\Routing\Dispatcher.php(167): Dispatcher->_invoke(Object(LutadoresController), Object(CakeRequest))
#4 C:\xampp\htdocs\rankings\index.php(117): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#5 {main}
2021-07-27 03:19:45 Error: [Error] Call to a member function find() on null
Request URL: /rankings/lutadores
Stack Trace:
#0 [internal function]: LutadoresController->index()
#1 C:\xampp\cake\lib\Cake\Controller\Controller.php(499): ReflectionMethod->invokeArgs(Object(LutadoresController), Array)
#2 C:\xampp\cake\lib\Cake\Routing\Dispatcher.php(193): Controller->invokeAction(Object(CakeRequest))
#3 C:\xampp\cake\lib\Cake\Routing\Dispatcher.php(167): Dispatcher->_invoke(Object(LutadoresController), Object(CakeRequest))
#4 C:\xampp\htdocs\rankings\index.php(117): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#5 {main}
2021-07-27 03:20:17 Error: [MissingViewException] View file "Lutador\index.ctp" is missing.
Exception Attributes: array (
  'file' => 'Lutador\\index.ctp',
)
Request URL: /rankings/lutadores
Stack Trace:
#0 C:\xampp\cake\lib\Cake\View\View.php(470): View->_getViewFileName('Lutador\\index')
#1 C:\xampp\cake\lib\Cake\Controller\Controller.php(968): View->render(NULL, NULL)
#2 C:\xampp\cake\lib\Cake\Routing\Dispatcher.php(200): Controller->render()
#3 C:\xampp\cake\lib\Cake\Routing\Dispatcher.php(167): Dispatcher->_invoke(Object(LutadoresController), Object(CakeRequest))
#4 C:\xampp\htdocs\rankings\index.php(117): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#5 {main}
2021-07-27 03:48:29 Error: [MissingControllerException] Controller class LutadoresController could not be found.
Exception Attributes: array (
  'class' => 'LutadoresController',
  'plugin' => NULL,
)
Request URL: /rankings/lutadores
Stack Trace:
#0 C:\xampp\htdocs\rankings\index.php(117): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#1 {main}
2021-07-27 03:48:35 Error: [MissingControllerException] Controller class LutadorsController could not be found.
Exception Attributes: array (
  'class' => 'LutadorsController',
  'plugin' => NULL,
)
Request URL: /rankings/lutadors
Stack Trace:
#0 C:\xampp\htdocs\rankings\index.php(117): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#1 {main}
2021-07-27 03:48:56 Error: [MissingViewException] View file "Lutadors\index.ctp" is missing.
Exception Attributes: array (
  'file' => 'Lutadors\\index.ctp',
)
Request URL: /rankings/lutadors
Stack Trace:
#0 C:\xampp\cake\lib\Cake\View\View.php(470): View->_getViewFileName('Lutadors\\index')
#1 C:\xampp\cake\lib\Cake\Controller\Controller.php(968): View->render(NULL, NULL)
#2 C:\xampp\cake\lib\Cake\Routing\Dispatcher.php(200): Controller->render()
#3 C:\xampp\cake\lib\Cake\Routing\Dispatcher.php(167): Dispatcher->_invoke(Object(LutadorsController), Object(CakeRequest))
#4 C:\xampp\htdocs\rankings\index.php(117): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#5 {main}
2021-07-27 19:12:37 Error: [MissingControllerException] Controller class LutarosController could not be found.
Exception Attributes: array (
  'class' => 'LutarosController',
  'plugin' => NULL,
)
Request URL: /ufc_rankings/lutaros
Stack Trace:
#0 C:\xampp\htdocs\ufc_rankings\index.php(117): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#1 {main}
2021-07-27 19:12:40 Error: [MissingControllerException] Controller class LutarsController could not be found.
Exception Attributes: array (
  'class' => 'LutarsController',
  'plugin' => NULL,
)
Request URL: /ufc_rankings/lutars
Stack Trace:
#0 C:\xampp\htdocs\ufc_rankings\index.php(117): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#1 {main}
2021-07-27 20:55:21 Error: [Error] Call to a member function diff() on string
Request URL: /ufc_rankings/lutadors
Stack Trace:
#0 C:\xampp\cake\lib\Cake\View\View.php(971): include()
#1 C:\xampp\cake\lib\Cake\View\View.php(933): View->_evaluate('C:\\xampp\\apps\\u...', Array)
#2 C:\xampp\cake\lib\Cake\View\View.php(473): View->_render('C:\\xampp\\apps\\u...')
#3 C:\xampp\cake\lib\Cake\Controller\Controller.php(968): View->render(NULL, NULL)
#4 C:\xampp\cake\lib\Cake\Routing\Dispatcher.php(200): Controller->render()
#5 C:\xampp\cake\lib\Cake\Routing\Dispatcher.php(167): Dispatcher->_invoke(Object(LutadorsController), Object(CakeRequest))
#6 C:\xampp\htdocs\ufc_rankings\index.php(117): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#7 {main}
2021-07-27 20:56:07 Error: [Error] Object of class DateInterval could not be converted to string
Request URL: /ufc_rankings/lutadors
Stack Trace:
#0 C:\xampp\cake\lib\Cake\View\Helper\HtmlHelper.php(945): sprintf('<td%s>%s</td>', '', Object(DateInterval))
#1 C:\xampp\apps\ufc_rankings\View\Lutadors\index.ctp(27): HtmlHelper->tableCells(Array)
#2 C:\xampp\cake\lib\Cake\View\View.php(971): include('C:\\xampp\\apps\\u...')
#3 C:\xampp\cake\lib\Cake\View\View.php(933): View->_evaluate('C:\\xampp\\apps\\u...', Array)
#4 C:\xampp\cake\lib\Cake\View\View.php(473): View->_render('C:\\xampp\\apps\\u...')
#5 C:\xampp\cake\lib\Cake\Controller\Controller.php(968): View->render(NULL, NULL)
#6 C:\xampp\cake\lib\Cake\Routing\Dispatcher.php(200): Controller->render()
#7 C:\xampp\cake\lib\Cake\Routing\Dispatcher.php(167): Dispatcher->_invoke(Object(LutadorsController), Object(CakeRequest))
#8 C:\xampp\htdocs\ufc_rankings\index.php(117): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#9 {main}
2021-07-27 20:56:53 Error: [Error] Object of class DateTime could not be converted to string
Request URL: /ufc_rankings/lutadors
Stack Trace:
#0 C:\xampp\cake\lib\Cake\View\View.php(971): include()
#1 C:\xampp\cake\lib\Cake\View\View.php(933): View->_evaluate('C:\\xampp\\apps\\u...', Array)
#2 C:\xampp\cake\lib\Cake\View\View.php(473): View->_render('C:\\xampp\\apps\\u...')
#3 C:\xampp\cake\lib\Cake\Controller\Controller.php(968): View->render(NULL, NULL)
#4 C:\xampp\cake\lib\Cake\Routing\Dispatcher.php(200): Controller->render()
#5 C:\xampp\cake\lib\Cake\Routing\Dispatcher.php(167): Dispatcher->_invoke(Object(LutadorsController), Object(CakeRequest))
#6 C:\xampp\htdocs\ufc_rankings\index.php(117): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#7 {main}
2021-07-27 20:57:10 Error: [Error] Object of class DateTime could not be converted to string
Request URL: /ufc_rankings/lutadors
Stack Trace:
#0 C:\xampp\cake\lib\Cake\View\View.php(971): include()
#1 C:\xampp\cake\lib\Cake\View\View.php(933): View->_evaluate('C:\\xampp\\apps\\u...', Array)
#2 C:\xampp\cake\lib\Cake\View\View.php(473): View->_render('C:\\xampp\\apps\\u...')
#3 C:\xampp\cake\lib\Cake\Controller\Controller.php(968): View->render(NULL, NULL)
#4 C:\xampp\cake\lib\Cake\Routing\Dispatcher.php(200): Controller->render()
#5 C:\xampp\cake\lib\Cake\Routing\Dispatcher.php(167): Dispatcher->_invoke(Object(LutadorsController), Object(CakeRequest))
#6 C:\xampp\htdocs\ufc_rankings\index.php(117): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#7 {main}
2021-07-27 21:21:54 Error: [Error] Class 'TimeZone' not found
Request URL: /ufc_rankings/lutadors
Stack Trace:
#0 C:\xampp\cake\lib\Cake\View\View.php(971): include()
#1 C:\xampp\cake\lib\Cake\View\View.php(933): View->_evaluate('C:\\xampp\\apps\\u...', Array)
#2 C:\xampp\cake\lib\Cake\View\View.php(473): View->_render('C:\\xampp\\apps\\u...')
#3 C:\xampp\cake\lib\Cake\Controller\Controller.php(968): View->render(NULL, NULL)
#4 C:\xampp\cake\lib\Cake\Routing\Dispatcher.php(200): Controller->render()
#5 C:\xampp\cake\lib\Cake\Routing\Dispatcher.php(167): Dispatcher->_invoke(Object(LutadorsController), Object(CakeRequest))
#6 C:\xampp\htdocs\ufc_rankings\index.php(117): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#7 {main}
2021-07-27 21:24:12 Error: [Error] Object of class DateInterval could not be converted to string
Request URL: /ufc_rankings/lutadors
Stack Trace:
#0 C:\xampp\cake\lib\Cake\View\View.php(971): include()
#1 C:\xampp\cake\lib\Cake\View\View.php(933): View->_evaluate('C:\\xampp\\apps\\u...', Array)
#2 C:\xampp\cake\lib\Cake\View\View.php(473): View->_render('C:\\xampp\\apps\\u...')
#3 C:\xampp\cake\lib\Cake\Controller\Controller.php(968): View->render(NULL, NULL)
#4 C:\xampp\cake\lib\Cake\Routing\Dispatcher.php(200): Controller->render()
#5 C:\xampp\cake\lib\Cake\Routing\Dispatcher.php(167): Dispatcher->_invoke(Object(LutadorsController), Object(CakeRequest))
#6 C:\xampp\htdocs\ufc_rankings\index.php(117): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#7 {main}
2021-07-27 21:24:22 Error: [Error] Object of class DateInterval could not be converted to string
Request URL: /ufc_rankings/lutadors
Stack Trace:
#0 C:\xampp\cake\lib\Cake\View\View.php(971): include()
#1 C:\xampp\cake\lib\Cake\View\View.php(933): View->_evaluate('C:\\xampp\\apps\\u...', Array)
#2 C:\xampp\cake\lib\Cake\View\View.php(473): View->_render('C:\\xampp\\apps\\u...')
#3 C:\xampp\cake\lib\Cake\Controller\Controller.php(968): View->render(NULL, NULL)
#4 C:\xampp\cake\lib\Cake\Routing\Dispatcher.php(200): Controller->render()
#5 C:\xampp\cake\lib\Cake\Routing\Dispatcher.php(167): Dispatcher->_invoke(Object(LutadorsController), Object(CakeRequest))
#6 C:\xampp\htdocs\ufc_rankings\index.php(117): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#7 {main}
2021-07-28 13:11:17 Error: [MissingControllerException] Controller class LutdorsController could not be found.
Exception Attributes: array (
  'class' => 'LutdorsController',
  'plugin' => NULL,
)
Request URL: /ufc_rankings/lutdors
Stack Trace:
#0 C:\xampp\htdocs\ufc_rankings\index.php(117): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#1 {main}
2021-07-28 13:16:11 Error: [MissingActionException] Action LutadorsController::edit() could not be found.
Exception Attributes: array (
  'controller' => 'LutadorsController',
  'action' => 'edit',
)
Request URL: /ufc_rankings/lutadors/edit/1
Stack Trace:
#0 C:\xampp\cake\lib\Cake\Routing\Dispatcher.php(193): Controller->invokeAction(Object(CakeRequest))
#1 C:\xampp\cake\lib\Cake\Routing\Dispatcher.php(167): Dispatcher->_invoke(Object(LutadorsController), Object(CakeRequest))
#2 C:\xampp\htdocs\ufc_rankings\index.php(117): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#3 {main}
2021-07-28 13:37:23 Error: [PDOException] SQLSTATE[23000]: Integrity constraint violation: 1062 Duplicate entry '2' for key 'PRIMARY'
Request URL: /ufc_rankings/lutadors/edit/2
Stack Trace:
#0 C:\xampp\cake\lib\Cake\Model\Datasource\DboSource.php(502): PDOStatement->execute(Array)
#1 C:\xampp\cake\lib\Cake\Model\Datasource\DboSource.php(468): DboSource->_execute('INSERT INTO `uf...', Array)
#2 C:\xampp\cake\lib\Cake\Model\Datasource\DboSource.php(1132): DboSource->execute('INSERT INTO `uf...')
#3 C:\xampp\cake\lib\Cake\Model\Model.php(1942): DboSource->create(Object(Lutador), Array, Array)
#4 C:\xampp\cake\lib\Cake\Model\Model.php(1760): Model->_doSave(Array, Array)
#5 C:\xampp\apps\ufc_rankings\Controller\LutadorsController.php(26): Model->save(Array)
#6 [internal function]: LutadorsController->edit('2')
#7 C:\xampp\cake\lib\Cake\Controller\Controller.php(499): ReflectionMethod->invokeArgs(Object(LutadorsController), Array)
#8 C:\xampp\cake\lib\Cake\Routing\Dispatcher.php(193): Controller->invokeAction(Object(CakeRequest))
#9 C:\xampp\cake\lib\Cake\Routing\Dispatcher.php(167): Dispatcher->_invoke(Object(LutadorsController), Object(CakeRequest))
#10 C:\xampp\htdocs\ufc_rankings\index.php(117): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#11 {main}
2021-07-28 13:38:36 Error: [PDOException] SQLSTATE[23000]: Integrity constraint violation: 1062 Duplicate entry '2' for key 'PRIMARY'
Request URL: /ufc_rankings/lutadors/edit/2
Stack Trace:
#0 C:\xampp\cake\lib\Cake\Model\Datasource\DboSource.php(502): PDOStatement->execute(Array)
#1 C:\xampp\cake\lib\Cake\Model\Datasource\DboSource.php(468): DboSource->_execute('INSERT INTO `uf...', Array)
#2 C:\xampp\cake\lib\Cake\Model\Datasource\DboSource.php(1132): DboSource->execute('INSERT INTO `uf...')
#3 C:\xampp\cake\lib\Cake\Model\Model.php(1942): DboSource->create(Object(Lutador), Array, Array)
#4 C:\xampp\cake\lib\Cake\Model\Model.php(1760): Model->_doSave(Array, Array)
#5 C:\xampp\apps\ufc_rankings\Controller\LutadorsController.php(26): Model->save(Array)
#6 [internal function]: LutadorsController->edit('2')
#7 C:\xampp\cake\lib\Cake\Controller\Controller.php(499): ReflectionMethod->invokeArgs(Object(LutadorsController), Array)
#8 C:\xampp\cake\lib\Cake\Routing\Dispatcher.php(193): Controller->invokeAction(Object(CakeRequest))
#9 C:\xampp\cake\lib\Cake\Routing\Dispatcher.php(167): Dispatcher->_invoke(Object(LutadorsController), Object(CakeRequest))
#10 C:\xampp\htdocs\ufc_rankings\index.php(117): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#11 {main}
2021-07-28 13:41:42 Error: [PDOException] SQLSTATE[23000]: Integrity constraint violation: 1062 Duplicate entry '1' for key 'PRIMARY'
Request URL: /ufc_rankings/lutadors/edit/1
Stack Trace:
#0 C:\xampp\cake\lib\Cake\Model\Datasource\DboSource.php(502): PDOStatement->execute(Array)
#1 C:\xampp\cake\lib\Cake\Model\Datasource\DboSource.php(468): DboSource->_execute('INSERT INTO `uf...', Array)
#2 C:\xampp\cake\lib\Cake\Model\Datasource\DboSource.php(1132): DboSource->execute('INSERT INTO `uf...')
#3 C:\xampp\cake\lib\Cake\Model\Model.php(1942): DboSource->create(Object(Lutador), Array, Array)
#4 C:\xampp\cake\lib\Cake\Model\Model.php(1760): Model->_doSave(Array, Array)
#5 C:\xampp\apps\ufc_rankings\Controller\LutadorsController.php(26): Model->save(Array)
#6 [internal function]: LutadorsController->edit('1')
#7 C:\xampp\cake\lib\Cake\Controller\Controller.php(499): ReflectionMethod->invokeArgs(Object(LutadorsController), Array)
#8 C:\xampp\cake\lib\Cake\Routing\Dispatcher.php(193): Controller->invokeAction(Object(CakeRequest))
#9 C:\xampp\cake\lib\Cake\Routing\Dispatcher.php(167): Dispatcher->_invoke(Object(LutadorsController), Object(CakeRequest))
#10 C:\xampp\htdocs\ufc_rankings\index.php(117): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#11 {main}
2021-07-28 13:47:05 Error: [PDOException] SQLSTATE[23000]: Integrity constraint violation: 1062 Duplicate entry '3' for key 'PRIMARY'
Request URL: /ufc_rankings/lutadors/edit/3
Stack Trace:
#0 C:\xampp\cake\lib\Cake\Model\Datasource\DboSource.php(502): PDOStatement->execute(Array)
#1 C:\xampp\cake\lib\Cake\Model\Datasource\DboSource.php(468): DboSource->_execute('INSERT INTO `uf...', Array)
#2 C:\xampp\cake\lib\Cake\Model\Datasource\DboSource.php(1132): DboSource->execute('INSERT INTO `uf...')
#3 C:\xampp\cake\lib\Cake\Model\Model.php(1942): DboSource->create(Object(Lutador), Array, Array)
#4 C:\xampp\cake\lib\Cake\Model\Model.php(1760): Model->_doSave(Array, Array)
#5 C:\xampp\apps\ufc_rankings\Controller\LutadorsController.php(26): Model->save(Array)
#6 [internal function]: LutadorsController->edit('3')
#7 C:\xampp\cake\lib\Cake\Controller\Controller.php(499): ReflectionMethod->invokeArgs(Object(LutadorsController), Array)
#8 C:\xampp\cake\lib\Cake\Routing\Dispatcher.php(193): Controller->invokeAction(Object(CakeRequest))
#9 C:\xampp\cake\lib\Cake\Routing\Dispatcher.php(167): Dispatcher->_invoke(Object(LutadorsController), Object(CakeRequest))
#10 C:\xampp\htdocs\ufc_rankings\index.php(117): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#11 {main}
2021-07-28 13:52:46 Error: [PDOException] SQLSTATE[23000]: Integrity constraint violation: 1062 Duplicate entry '1' for key 'PRIMARY'
Request URL: /ufc_rankings/lutadors/edit/1
Stack Trace:
#0 C:\xampp\cake\lib\Cake\Model\Datasource\DboSource.php(502): PDOStatement->execute(Array)
#1 C:\xampp\cake\lib\Cake\Model\Datasource\DboSource.php(468): DboSource->_execute('INSERT INTO `uf...', Array)
#2 C:\xampp\cake\lib\Cake\Model\Datasource\DboSource.php(1132): DboSource->execute('INSERT INTO `uf...')
#3 C:\xampp\cake\lib\Cake\Model\Model.php(1942): DboSource->create(Object(Lutador), Array, Array)
#4 C:\xampp\cake\lib\Cake\Model\Model.php(1760): Model->_doSave(Array, Array)
#5 C:\xampp\apps\ufc_rankings\Controller\LutadorsController.php(26): Model->save(Array)
#6 [internal function]: LutadorsController->edit('1')
#7 C:\xampp\cake\lib\Cake\Controller\Controller.php(499): ReflectionMethod->invokeArgs(Object(LutadorsController), Array)
#8 C:\xampp\cake\lib\Cake\Routing\Dispatcher.php(193): Controller->invokeAction(Object(CakeRequest))
#9 C:\xampp\cake\lib\Cake\Routing\Dispatcher.php(167): Dispatcher->_invoke(Object(LutadorsController), Object(CakeRequest))
#10 C:\xampp\htdocs\ufc_rankings\index.php(117): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#11 {main}
2021-07-28 13:54:23 Error: [PDOException] SQLSTATE[23000]: Integrity constraint violation: 1062 Duplicate entry '1' for key 'PRIMARY'
Request URL: /ufc_rankings/lutadors/edit/1
Stack Trace:
#0 C:\xampp\cake\lib\Cake\Model\Datasource\DboSource.php(502): PDOStatement->execute(Array)
#1 C:\xampp\cake\lib\Cake\Model\Datasource\DboSource.php(468): DboSource->_execute('INSERT INTO `uf...', Array)
#2 C:\xampp\cake\lib\Cake\Model\Datasource\DboSource.php(1132): DboSource->execute('INSERT INTO `uf...')
#3 C:\xampp\cake\lib\Cake\Model\Model.php(1942): DboSource->create(Object(Lutador), Array, Array)
#4 C:\xampp\cake\lib\Cake\Model\Model.php(1760): Model->_doSave(Array, Array)
#5 C:\xampp\apps\ufc_rankings\Controller\LutadorsController.php(27): Model->save(Array)
#6 [internal function]: LutadorsController->edit('1')
#7 C:\xampp\cake\lib\Cake\Controller\Controller.php(499): ReflectionMethod->invokeArgs(Object(LutadorsController), Array)
#8 C:\xampp\cake\lib\Cake\Routing\Dispatcher.php(193): Controller->invokeAction(Object(CakeRequest))
#9 C:\xampp\cake\lib\Cake\Routing\Dispatcher.php(167): Dispatcher->_invoke(Object(LutadorsController), Object(CakeRequest))
#10 C:\xampp\htdocs\ufc_rankings\index.php(117): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#11 {main}
2021-07-28 13:56:28 Error: [PDOException] SQLSTATE[23000]: Integrity constraint violation: 1062 Duplicate entry '1' for key 'PRIMARY'
Request URL: /ufc_rankings/lutadors/edit/1
Stack Trace:
#0 C:\xampp\cake\lib\Cake\Model\Datasource\DboSource.php(502): PDOStatement->execute(Array)
#1 C:\xampp\cake\lib\Cake\Model\Datasource\DboSource.php(468): DboSource->_execute('INSERT INTO `uf...', Array)
#2 C:\xampp\cake\lib\Cake\Model\Datasource\DboSource.php(1132): DboSource->execute('INSERT INTO `uf...')
#3 C:\xampp\cake\lib\Cake\Model\Model.php(1942): DboSource->create(Object(Lutador), Array, Array)
#4 C:\xampp\cake\lib\Cake\Model\Model.php(1760): Model->_doSave(Array, Array)
#5 C:\xampp\apps\ufc_rankings\Controller\LutadorsController.php(27): Model->save(Array)
#6 [internal function]: LutadorsController->edit('1')
#7 C:\xampp\cake\lib\Cake\Controller\Controller.php(499): ReflectionMethod->invokeArgs(Object(LutadorsController), Array)
#8 C:\xampp\cake\lib\Cake\Routing\Dispatcher.php(193): Controller->invokeAction(Object(CakeRequest))
#9 C:\xampp\cake\lib\Cake\Routing\Dispatcher.php(167): Dispatcher->_invoke(Object(LutadorsController), Object(CakeRequest))
#10 C:\xampp\htdocs\ufc_rankings\index.php(117): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#11 {main}
2021-07-28 13:56:46 Error: [PDOException] SQLSTATE[23000]: Integrity constraint violation: 1062 Duplicate entry '1' for key 'PRIMARY'
Request URL: /ufc_rankings/lutadors/edit/1
Stack Trace:
#0 C:\xampp\cake\lib\Cake\Model\Datasource\DboSource.php(502): PDOStatement->execute(Array)
#1 C:\xampp\cake\lib\Cake\Model\Datasource\DboSource.php(468): DboSource->_execute('INSERT INTO `uf...', Array)
#2 C:\xampp\cake\lib\Cake\Model\Datasource\DboSource.php(1132): DboSource->execute('INSERT INTO `uf...')
#3 C:\xampp\cake\lib\Cake\Model\Model.php(1942): DboSource->create(Object(Lutador), Array, Array)
#4 C:\xampp\cake\lib\Cake\Model\Model.php(1760): Model->_doSave(Array, Array)
#5 C:\xampp\apps\ufc_rankings\Controller\LutadorsController.php(27): Model->save(Array)
#6 [internal function]: LutadorsController->edit('1')
#7 C:\xampp\cake\lib\Cake\Controller\Controller.php(499): ReflectionMethod->invokeArgs(Object(LutadorsController), Array)
#8 C:\xampp\cake\lib\Cake\Routing\Dispatcher.php(193): Controller->invokeAction(Object(CakeRequest))
#9 C:\xampp\cake\lib\Cake\Routing\Dispatcher.php(167): Dispatcher->_invoke(Object(LutadorsController), Object(CakeRequest))
#10 C:\xampp\htdocs\ufc_rankings\index.php(117): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#11 {main}
2021-07-28 13:56:49 Error: [PDOException] SQLSTATE[23000]: Integrity constraint violation: 1062 Duplicate entry '1' for key 'PRIMARY'
Request URL: /ufc_rankings/lutadors/edit/1
Stack Trace:
#0 C:\xampp\cake\lib\Cake\Model\Datasource\DboSource.php(502): PDOStatement->execute(Array)
#1 C:\xampp\cake\lib\Cake\Model\Datasource\DboSource.php(468): DboSource->_execute('INSERT INTO `uf...', Array)
#2 C:\xampp\cake\lib\Cake\Model\Datasource\DboSource.php(1132): DboSource->execute('INSERT INTO `uf...')
#3 C:\xampp\cake\lib\Cake\Model\Model.php(1942): DboSource->create(Object(Lutador), Array, Array)
#4 C:\xampp\cake\lib\Cake\Model\Model.php(1760): Model->_doSave(Array, Array)
#5 C:\xampp\apps\ufc_rankings\Controller\LutadorsController.php(27): Model->save(Array)
#6 [internal function]: LutadorsController->edit('1')
#7 C:\xampp\cake\lib\Cake\Controller\Controller.php(499): ReflectionMethod->invokeArgs(Object(LutadorsController), Array)
#8 C:\xampp\cake\lib\Cake\Routing\Dispatcher.php(193): Controller->invokeAction(Object(CakeRequest))
#9 C:\xampp\cake\lib\Cake\Routing\Dispatcher.php(167): Dispatcher->_invoke(Object(LutadorsController), Object(CakeRequest))
#10 C:\xampp\htdocs\ufc_rankings\index.php(117): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#11 {main}
2021-07-28 13:56:55 Error: [PDOException] SQLSTATE[23000]: Integrity constraint violation: 1062 Duplicate entry '1' for key 'PRIMARY'
Request URL: /ufc_rankings/lutadors/edit/1
Stack Trace:
#0 C:\xampp\cake\lib\Cake\Model\Datasource\DboSource.php(502): PDOStatement->execute(Array)
#1 C:\xampp\cake\lib\Cake\Model\Datasource\DboSource.php(468): DboSource->_execute('INSERT INTO `uf...', Array)
#2 C:\xampp\cake\lib\Cake\Model\Datasource\DboSource.php(1132): DboSource->execute('INSERT INTO `uf...')
#3 C:\xampp\cake\lib\Cake\Model\Model.php(1942): DboSource->create(Object(Lutador), Array, Array)
#4 C:\xampp\cake\lib\Cake\Model\Model.php(1760): Model->_doSave(Array, Array)
#5 C:\xampp\apps\ufc_rankings\Controller\LutadorsController.php(27): Model->save(Array)
#6 [internal function]: LutadorsController->edit('1')
#7 C:\xampp\cake\lib\Cake\Controller\Controller.php(499): ReflectionMethod->invokeArgs(Object(LutadorsController), Array)
#8 C:\xampp\cake\lib\Cake\Routing\Dispatcher.php(193): Controller->invokeAction(Object(CakeRequest))
#9 C:\xampp\cake\lib\Cake\Routing\Dispatcher.php(167): Dispatcher->_invoke(Object(LutadorsController), Object(CakeRequest))
#10 C:\xampp\htdocs\ufc_rankings\index.php(117): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#11 {main}
2021-07-28 13:57:46 Error: [PDOException] SQLSTATE[23000]: Integrity constraint violation: 1062 Duplicate entry '1' for key 'PRIMARY'
Request URL: /ufc_rankings/lutadors/edit/1
Stack Trace:
#0 C:\xampp\cake\lib\Cake\Model\Datasource\DboSource.php(502): PDOStatement->execute(Array)
#1 C:\xampp\cake\lib\Cake\Model\Datasource\DboSource.php(468): DboSource->_execute('INSERT INTO `uf...', Array)
#2 C:\xampp\cake\lib\Cake\Model\Datasource\DboSource.php(1132): DboSource->execute('INSERT INTO `uf...')
#3 C:\xampp\cake\lib\Cake\Model\Model.php(1942): DboSource->create(Object(Lutador), Array, Array)
#4 C:\xampp\cake\lib\Cake\Model\Model.php(1754): Model->_doSave(Array, Array)
#5 C:\xampp\cake\lib\Cake\Model\Model.php(2515): Model->save(Array, Array)
#6 C:\xampp\cake\lib\Cake\Model\Model.php(2287): Model->saveAssociated(Array, Array)
#7 C:\xampp\apps\ufc_rankings\Controller\LutadorsController.php(27): Model->saveAll(Array)
#8 [internal function]: LutadorsController->edit('1')
#9 C:\xampp\cake\lib\Cake\Controller\Controller.php(499): ReflectionMethod->invokeArgs(Object(LutadorsController), Array)
#10 C:\xampp\cake\lib\Cake\Routing\Dispatcher.php(193): Controller->invokeAction(Object(CakeRequest))
#11 C:\xampp\cake\lib\Cake\Routing\Dispatcher.php(167): Dispatcher->_invoke(Object(LutadorsController), Object(CakeRequest))
#12 C:\xampp\htdocs\ufc_rankings\index.php(117): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#13 {main}
2021-07-28 13:57:54 Error: [PDOException] SQLSTATE[23000]: Integrity constraint violation: 1062 Duplicate entry '1' for key 'PRIMARY'
Request URL: /ufc_rankings/lutadors/edit/1
Stack Trace:
#0 C:\xampp\cake\lib\Cake\Model\Datasource\DboSource.php(502): PDOStatement->execute(Array)
#1 C:\xampp\cake\lib\Cake\Model\Datasource\DboSource.php(468): DboSource->_execute('INSERT INTO `uf...', Array)
#2 C:\xampp\cake\lib\Cake\Model\Datasource\DboSource.php(1132): DboSource->execute('INSERT INTO `uf...')
#3 C:\xampp\cake\lib\Cake\Model\Model.php(1942): DboSource->create(Object(Lutador), Array, Array)
#4 C:\xampp\cake\lib\Cake\Model\Model.php(1754): Model->_doSave(Array, Array)
#5 C:\xampp\cake\lib\Cake\Model\Model.php(2515): Model->save(Array, Array)
#6 C:\xampp\cake\lib\Cake\Model\Model.php(2287): Model->saveAssociated(Array, Array)
#7 C:\xampp\apps\ufc_rankings\Controller\LutadorsController.php(27): Model->saveAll(Array)
#8 [internal function]: LutadorsController->edit('1')
#9 C:\xampp\cake\lib\Cake\Controller\Controller.php(499): ReflectionMethod->invokeArgs(Object(LutadorsController), Array)
#10 C:\xampp\cake\lib\Cake\Routing\Dispatcher.php(193): Controller->invokeAction(Object(CakeRequest))
#11 C:\xampp\cake\lib\Cake\Routing\Dispatcher.php(167): Dispatcher->_invoke(Object(LutadorsController), Object(CakeRequest))
#12 C:\xampp\htdocs\ufc_rankings\index.php(117): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#13 {main}
2021-07-28 13:58:51 Error: [PDOException] SQLSTATE[23000]: Integrity constraint violation: 1062 Duplicate entry '1' for key 'PRIMARY'
Request URL: /ufc_rankings/lutadors/edit/1
Stack Trace:
#0 C:\xampp\cake\lib\Cake\Model\Datasource\DboSource.php(502): PDOStatement->execute(Array)
#1 C:\xampp\cake\lib\Cake\Model\Datasource\DboSource.php(468): DboSource->_execute('INSERT INTO `uf...', Array)
#2 C:\xampp\cake\lib\Cake\Model\Datasource\DboSource.php(1132): DboSource->execute('INSERT INTO `uf...')
#3 C:\xampp\cake\lib\Cake\Model\Model.php(1942): DboSource->create(Object(Lutador), Array, Array)
#4 C:\xampp\cake\lib\Cake\Model\Model.php(1754): Model->_doSave(Array, Array)
#5 C:\xampp\cake\lib\Cake\Model\Model.php(2515): Model->save(Array, Array)
#6 C:\xampp\cake\lib\Cake\Model\Model.php(2287): Model->saveAssociated(Array, Array)
#7 C:\xampp\apps\ufc_rankings\Controller\LutadorsController.php(27): Model->saveAll(Array)
#8 [internal function]: LutadorsController->edit('1')
#9 C:\xampp\cake\lib\Cake\Controller\Controller.php(499): ReflectionMethod->invokeArgs(Object(LutadorsController), Array)
#10 C:\xampp\cake\lib\Cake\Routing\Dispatcher.php(193): Controller->invokeAction(Object(CakeRequest))
#11 C:\xampp\cake\lib\Cake\Routing\Dispatcher.php(167): Dispatcher->_invoke(Object(LutadorsController), Object(CakeRequest))
#12 C:\xampp\htdocs\ufc_rankings\index.php(117): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#13 {main}
2021-07-28 13:58:59 Error: [PDOException] SQLSTATE[23000]: Integrity constraint violation: 1062 Duplicate entry '3' for key 'PRIMARY'
Request URL: /ufc_rankings/lutadors/edit/3
Stack Trace:
#0 C:\xampp\cake\lib\Cake\Model\Datasource\DboSource.php(502): PDOStatement->execute(Array)
#1 C:\xampp\cake\lib\Cake\Model\Datasource\DboSource.php(468): DboSource->_execute('INSERT INTO `uf...', Array)
#2 C:\xampp\cake\lib\Cake\Model\Datasource\DboSource.php(1132): DboSource->execute('INSERT INTO `uf...')
#3 C:\xampp\cake\lib\Cake\Model\Model.php(1942): DboSource->create(Object(Lutador), Array, Array)
#4 C:\xampp\cake\lib\Cake\Model\Model.php(1754): Model->_doSave(Array, Array)
#5 C:\xampp\cake\lib\Cake\Model\Model.php(2515): Model->save(Array, Array)
#6 C:\xampp\cake\lib\Cake\Model\Model.php(2287): Model->saveAssociated(Array, Array)
#7 C:\xampp\apps\ufc_rankings\Controller\LutadorsController.php(27): Model->saveAll(Array)
#8 [internal function]: LutadorsController->edit('3')
#9 C:\xampp\cake\lib\Cake\Controller\Controller.php(499): ReflectionMethod->invokeArgs(Object(LutadorsController), Array)
#10 C:\xampp\cake\lib\Cake\Routing\Dispatcher.php(193): Controller->invokeAction(Object(CakeRequest))
#11 C:\xampp\cake\lib\Cake\Routing\Dispatcher.php(167): Dispatcher->_invoke(Object(LutadorsController), Object(CakeRequest))
#12 C:\xampp\htdocs\ufc_rankings\index.php(117): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#13 {main}
2021-07-28 14:02:12 Error: [PDOException] SQLSTATE[23000]: Integrity constraint violation: 1062 Duplicate entry '1' for key 'PRIMARY'
Request URL: /ufc_rankings/lutadors/edit/1
Stack Trace:
#0 C:\xampp\cake\lib\Cake\Model\Datasource\DboSource.php(502): PDOStatement->execute(Array)
#1 C:\xampp\cake\lib\Cake\Model\Datasource\DboSource.php(468): DboSource->_execute('INSERT INTO `uf...', Array)
#2 C:\xampp\cake\lib\Cake\Model\Datasource\DboSource.php(1132): DboSource->execute('INSERT INTO `uf...')
#3 C:\xampp\cake\lib\Cake\Model\Model.php(1942): DboSource->create(Object(Lutador), Array, Array)
#4 C:\xampp\cake\lib\Cake\Model\Model.php(1760): Model->_doSave(Array, Array)
#5 C:\xampp\apps\ufc_rankings\Controller\LutadorsController.php(27): Model->save(Array)
#6 [internal function]: LutadorsController->edit('1')
#7 C:\xampp\cake\lib\Cake\Controller\Controller.php(499): ReflectionMethod->invokeArgs(Object(LutadorsController), Array)
#8 C:\xampp\cake\lib\Cake\Routing\Dispatcher.php(193): Controller->invokeAction(Object(CakeRequest))
#9 C:\xampp\cake\lib\Cake\Routing\Dispatcher.php(167): Dispatcher->_invoke(Object(LutadorsController), Object(CakeRequest))
#10 C:\xampp\htdocs\ufc_rankings\index.php(117): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#11 {main}
2021-07-28 14:02:41 Error: [PDOException] SQLSTATE[23000]: Integrity constraint violation: 1062 Duplicate entry '1' for key 'PRIMARY'
Request URL: /ufc_rankings/lutadors/edit/1
Stack Trace:
#0 C:\xampp\cake\lib\Cake\Model\Datasource\DboSource.php(502): PDOStatement->execute(Array)
#1 C:\xampp\cake\lib\Cake\Model\Datasource\DboSource.php(468): DboSource->_execute('INSERT INTO `uf...', Array)
#2 C:\xampp\cake\lib\Cake\Model\Datasource\DboSource.php(1132): DboSource->execute('INSERT INTO `uf...')
#3 C:\xampp\cake\lib\Cake\Model\Model.php(1942): DboSource->create(Object(Lutador), Array, Array)
#4 C:\xampp\cake\lib\Cake\Model\Model.php(1760): Model->_doSave(Array, Array)
#5 C:\xampp\apps\ufc_rankings\Controller\LutadorsController.php(27): Model->save(Array)
#6 [internal function]: LutadorsController->edit('1')
#7 C:\xampp\cake\lib\Cake\Controller\Controller.php(499): ReflectionMethod->invokeArgs(Object(LutadorsController), Array)
#8 C:\xampp\cake\lib\Cake\Routing\Dispatcher.php(193): Controller->invokeAction(Object(CakeRequest))
#9 C:\xampp\cake\lib\Cake\Routing\Dispatcher.php(167): Dispatcher->_invoke(Object(LutadorsController), Object(CakeRequest))
#10 C:\xampp\htdocs\ufc_rankings\index.php(117): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#11 {main}
2021-07-28 14:05:16 Error: [PDOException] SQLSTATE[23000]: Integrity constraint violation: 1062 Duplicate entry '1' for key 'PRIMARY'
Request URL: /ufc_rankings/lutadors/edit/1
Stack Trace:
#0 C:\xampp\cake\lib\Cake\Model\Datasource\DboSource.php(502): PDOStatement->execute(Array)
#1 C:\xampp\cake\lib\Cake\Model\Datasource\DboSource.php(468): DboSource->_execute('INSERT INTO `uf...', Array)
#2 C:\xampp\cake\lib\Cake\Model\Datasource\DboSource.php(1132): DboSource->execute('INSERT INTO `uf...')
#3 C:\xampp\cake\lib\Cake\Model\Model.php(1942): DboSource->create(Object(Lutador), Array, Array)
#4 C:\xampp\cake\lib\Cake\Model\Model.php(1760): Model->_doSave(Array, Array)
#5 C:\xampp\apps\ufc_rankings\Controller\LutadorsController.php(27): Model->save(Array)
#6 [internal function]: LutadorsController->edit('1')
#7 C:\xampp\cake\lib\Cake\Controller\Controller.php(499): ReflectionMethod->invokeArgs(Object(LutadorsController), Array)
#8 C:\xampp\cake\lib\Cake\Routing\Dispatcher.php(193): Controller->invokeAction(Object(CakeRequest))
#9 C:\xampp\cake\lib\Cake\Routing\Dispatcher.php(167): Dispatcher->_invoke(Object(LutadorsController), Object(CakeRequest))
#10 C:\xampp\htdocs\ufc_rankings\index.php(117): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#11 {main}
2021-07-28 14:05:29 Error: [PDOException] SQLSTATE[23000]: Integrity constraint violation: 1062 Duplicate entry '1' for key 'PRIMARY'
Request URL: /ufc_rankings/lutadors/edit/1
Stack Trace:
#0 C:\xampp\cake\lib\Cake\Model\Datasource\DboSource.php(502): PDOStatement->execute(Array)
#1 C:\xampp\cake\lib\Cake\Model\Datasource\DboSource.php(468): DboSource->_execute('INSERT INTO `uf...', Array)
#2 C:\xampp\cake\lib\Cake\Model\Datasource\DboSource.php(1132): DboSource->execute('INSERT INTO `uf...')
#3 C:\xampp\cake\lib\Cake\Model\Model.php(1942): DboSource->create(Object(Lutador), Array, Array)
#4 C:\xampp\cake\lib\Cake\Model\Model.php(1760): Model->_doSave(Array, Array)
#5 C:\xampp\apps\ufc_rankings\Controller\LutadorsController.php(27): Model->save(Array)
#6 [internal function]: LutadorsController->edit('1')
#7 C:\xampp\cake\lib\Cake\Controller\Controller.php(499): ReflectionMethod->invokeArgs(Object(LutadorsController), Array)
#8 C:\xampp\cake\lib\Cake\Routing\Dispatcher.php(193): Controller->invokeAction(Object(CakeRequest))
#9 C:\xampp\cake\lib\Cake\Routing\Dispatcher.php(167): Dispatcher->_invoke(Object(LutadorsController), Object(CakeRequest))
#10 C:\xampp\htdocs\ufc_rankings\index.php(117): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#11 {main}
2021-07-28 14:05:40 Error: [PDOException] SQLSTATE[23000]: Integrity constraint violation: 1062 Duplicate entry '6' for key 'PRIMARY'
Request URL: /ufc_rankings/lutadors/edit/6
Stack Trace:
#0 C:\xampp\cake\lib\Cake\Model\Datasource\DboSource.php(502): PDOStatement->execute(Array)
#1 C:\xampp\cake\lib\Cake\Model\Datasource\DboSource.php(468): DboSource->_execute('INSERT INTO `uf...', Array)
#2 C:\xampp\cake\lib\Cake\Model\Datasource\DboSource.php(1132): DboSource->execute('INSERT INTO `uf...')
#3 C:\xampp\cake\lib\Cake\Model\Model.php(1942): DboSource->create(Object(Lutador), Array, Array)
#4 C:\xampp\cake\lib\Cake\Model\Model.php(1760): Model->_doSave(Array, Array)
#5 C:\xampp\apps\ufc_rankings\Controller\LutadorsController.php(27): Model->save(Array)
#6 [internal function]: LutadorsController->edit('6')
#7 C:\xampp\cake\lib\Cake\Controller\Controller.php(499): ReflectionMethod->invokeArgs(Object(LutadorsController), Array)
#8 C:\xampp\cake\lib\Cake\Routing\Dispatcher.php(193): Controller->invokeAction(Object(CakeRequest))
#9 C:\xampp\cake\lib\Cake\Routing\Dispatcher.php(167): Dispatcher->_invoke(Object(LutadorsController), Object(CakeRequest))
#10 C:\xampp\htdocs\ufc_rankings\index.php(117): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#11 {main}
2021-07-28 14:12:52 Error: [PDOException] SQLSTATE[23000]: Integrity constraint violation: 1062 Duplicate entry '7' for key 'PRIMARY'
Request URL: /ufc_rankings/lutadors/edit/7
Stack Trace:
#0 C:\xampp\cake\lib\Cake\Model\Datasource\DboSource.php(502): PDOStatement->execute(Array)
#1 C:\xampp\cake\lib\Cake\Model\Datasource\DboSource.php(468): DboSource->_execute('INSERT INTO `uf...', Array)
#2 C:\xampp\cake\lib\Cake\Model\Datasource\DboSource.php(1132): DboSource->execute('INSERT INTO `uf...')
#3 C:\xampp\cake\lib\Cake\Model\Model.php(1942): DboSource->create(Object(Lutador), Array, Array)
#4 C:\xampp\cake\lib\Cake\Model\Model.php(1760): Model->_doSave(Array, Array)
#5 C:\xampp\apps\ufc_rankings\Controller\LutadorsController.php(27): Model->save(Array)
#6 [internal function]: LutadorsController->edit('7')
#7 C:\xampp\cake\lib\Cake\Controller\Controller.php(499): ReflectionMethod->invokeArgs(Object(LutadorsController), Array)
#8 C:\xampp\cake\lib\Cake\Routing\Dispatcher.php(193): Controller->invokeAction(Object(CakeRequest))
#9 C:\xampp\cake\lib\Cake\Routing\Dispatcher.php(167): Dispatcher->_invoke(Object(LutadorsController), Object(CakeRequest))
#10 C:\xampp\htdocs\ufc_rankings\index.php(117): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#11 {main}
2021-07-28 15:18:12 Error: [MissingActionException] Action LutadorsController::view1() could not be found.
Exception Attributes: array (
  'controller' => 'LutadorsController',
  'action' => 'view1',
)
Request URL: /ufc_rankings/lutadors/view1
Stack Trace:
#0 C:\xampp\cake\lib\Cake\Routing\Dispatcher.php(193): Controller->invokeAction(Object(CakeRequest))
#1 C:\xampp\cake\lib\Cake\Routing\Dispatcher.php(167): Dispatcher->_invoke(Object(LutadorsController), Object(CakeRequest))
#2 C:\xampp\htdocs\ufc_rankings\index.php(117): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#3 {main}
2021-07-28 15:20:33 Error: [MissingActionException] Action LutadorsController::view1() could not be found.
Exception Attributes: array (
  'controller' => 'LutadorsController',
  'action' => 'view1',
)
Request URL: /ufc_rankings/lutadors/view1
Stack Trace:
#0 C:\xampp\cake\lib\Cake\Routing\Dispatcher.php(193): Controller->invokeAction(Object(CakeRequest))
#1 C:\xampp\cake\lib\Cake\Routing\Dispatcher.php(167): Dispatcher->_invoke(Object(LutadorsController), Object(CakeRequest))
#2 C:\xampp\htdocs\ufc_rankings\index.php(117): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#3 {main}
2021-07-28 15:21:29 Error: [MissingActionException] Action LutadorsController::view1() could not be found.
Exception Attributes: array (
  'controller' => 'LutadorsController',
  'action' => 'view1',
)
Request URL: /ufc_rankings/lutadors/view1
Stack Trace:
#0 C:\xampp\cake\lib\Cake\Routing\Dispatcher.php(193): Controller->invokeAction(Object(CakeRequest))
#1 C:\xampp\cake\lib\Cake\Routing\Dispatcher.php(167): Dispatcher->_invoke(Object(LutadorsController), Object(CakeRequest))
#2 C:\xampp\htdocs\ufc_rankings\index.php(117): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#3 {main}
2021-07-28 15:21:32 Error: [MissingActionException] Action LutadorsController::view1() could not be found.
Exception Attributes: array (
  'controller' => 'LutadorsController',
  'action' => 'view1',
)
Request URL: /ufc_rankings/lutadors/view1
Stack Trace:
#0 C:\xampp\cake\lib\Cake\Routing\Dispatcher.php(193): Controller->invokeAction(Object(CakeRequest))
#1 C:\xampp\cake\lib\Cake\Routing\Dispatcher.php(167): Dispatcher->_invoke(Object(LutadorsController), Object(CakeRequest))
#2 C:\xampp\htdocs\ufc_rankings\index.php(117): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#3 {main}
2021-07-28 15:22:18 Error: [MissingActionException] Action LutadorsController::lutadors() could not be found.
Exception Attributes: array (
  'controller' => 'LutadorsController',
  'action' => 'lutadors',
)
Request URL: /ufc_rankings/lutadors/lutadors/view/1
Stack Trace:
#0 C:\xampp\cake\lib\Cake\Routing\Dispatcher.php(193): Controller->invokeAction(Object(CakeRequest))
#1 C:\xampp\cake\lib\Cake\Routing\Dispatcher.php(167): Dispatcher->_invoke(Object(LutadorsController), Object(CakeRequest))
#2 C:\xampp\htdocs\ufc_rankings\index.php(117): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#3 {main}
